pkg/utils/build/quay.go

```diff
@@ -31,6 +31,16 @@ func GetQuayImageName(annotations map[string]string) (string, error) {
 	return strings.Join(tokens[2:], "/"), nil
 }
 
+func IsImageAnnotationPresent(annotations map[string]string) bool {
+	image_annotation_str := annotations["image.redhat.com/image"]
+	return image_annotation_str != ""
+}
+
+func ImageAnnotationGenerateValueIsNotFailed(annotations map[string]string) bool {
+	generate_value_str := annotations["image.redhat.com/generate"]
```

Comment:
should we also check that the `image.redhat.com/generate` annotation exists, i.e. is not empty?

Reply:
Added check to see if the value is empty.

Reply:
I mean this one https://github.com/redhat-appstudio/image-controller/pull/33 where 'image.redhat.com/generate' is deleted after processed.

Reply:
Okay got it.. Removed the empty value check in the latest commit d7032bf818d4e603b2bf7d0fbe6f17e8dd316856